### PR TITLE
Adds taxi cleanup code

### PR DESCRIPTION
**Pull request by sarahdudek**
Python file for cleanup up the taxi trip csv file.

@minhahoang @ltyomkin @LuisFlosi

### CODE REVIEW

File: Cleaning_Code/taxi_cleanup.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                "pickup centroid location",
+                "dropoff centroid latitude",
+                "dropoff centroid longitude",
+                "dropoff centroid location",
+                "start_segment_id", 
+                "end_segment_id"]
+
+now = datetime.now()
+
+def hour_rounder(t):
```
Comment (sarahdudek) [inline Cleaning_Code/taxi_cleanup.py:53]: @LuisFlosi Grabbed this off of [stackoverflow](https://stackoverflow.com/questions/48937900/round-time-to-nearest-hour-python)

What were you thinking of using for this solution?

File: Cleaning_Code/taxi_cleanup.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+           "extra_charges", 
+           "trip_total", 
+           "payment_type"]
+
+columns_to_add = ["ride_type_id",
+                  "start_segment_id", 
+                  "end_segment_id"] 
+
+taxi_trip_df.drop(columns_to_delete, axis = 1)
+taxi_trip_df = new_column_names
```
Comment (LuisFlosi) [inline Cleaning_Code/taxi_cleanup.py]: Will this substitute the column names? I though we had to call taxi_trip_df.columns for it to work

File: Cleaning_Code/taxi_cleanup.py (hunk 1 above)
Comment (LuisFlosi) [reply Cleaning_Code/taxi_cleanup.py:53]: If I understand correctly, you are adding half hours to this? It should be only hour to hour

Comment (LuisFlosi) [reply Cleaning_Code/taxi_cleanup.py:53]: Also I don't get why we are creating the variable now?

**Comment by sarahdudek** [comment]
@LuisFlosi This is a comment

**Comment by LuisFlosi** [comment]
@sarahdudek are we good to merge this?

**Pull request merged**